src/schema_formats/XML/GMP.xml.in

```diff
@@ -27205,4 +27205,52 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
      GET_TASKS  - RESULT_COUNT
   -->
 
+  <!-- Deprecation warnings. -->
+
+  <deprecation>
+    <command>COMMANDS</command>
+    <summary>The command COMMANDS will be removed.</summary>
+    <description>
+      <p>
+        The command COMMANDS was used to send multiple commands in one request.
+        This was meant to avoid multiple server-roundtrips. Since the rountrips
+        are not expensive anymore, GMP clients should send each command in
+        a separate request.
+      </p>
+    </description>
+    <version>8.0</version>
+  </deprecation>
+
+  <deprecation>
+    <command>CREATE_REPORT</command>
+    <summary>Task creation will be removed.</summary>
+    <description>
+      <p>
+        The command CREATE_REPORT used to offer element TASK/NAME and TASK/COMMENT
+        in order to automactically let create a new task for the imported report.
+        This functionality is going to be removed. Instead use the direct command CREATE_TASK
+        prior to CREATE_REPORT.
+      </p>
+    </description>
+    <version>8.0</version>
+  </deprecation>
+
+  <deprecation>
+    <command>GET_REPORTS</command>
+    <summary>Drop the attributes TYPE, HOST, HOST_FIRST_RESULT, HOST_MAX_RESULTS, HOST_LEVELS, HOST_SEARCH_PHRASE and POS.</summary>
+    <description>
+      <p>
+        In the current version, TYPE can be either SCAN or ASSETS and it defaults to SCAN.
+        ASSETS will be removed, because GET_ASSETS should be used instead.
+        If you are using TYPE with SCAN, simply remove this part, because of the
+        equavalent default.
```

Comment:
```suggestion
        is always scan.
```

Might be better.  I didn't understand the sentence so easily before.

Reply:
Also, I don't normally uppercase attributes, but use this: attribute "type".  But not a big deal.